mastergateway/mastergateway.go

```diff
@@ -31,6 +31,7 @@ type config struct {
 	OwnOrganization    string                           // The name of the own organization (Should also be present in the list "Organizations")
 	StorageURI         string                           // URI of HolmesStorage
 	AutoTasks          map[string](map[string][]string) // Tasks that should be automatically executed on new objects mimetype -> taskname -> args
+	MaxUploadSize      int64                            // The maximum size of a sample-upload in Megabyte
```

Comment:
Actually, you are right. I chose int64 because the bodie's content length is stored as an int64 and it wouldn't have let me compare these without typecasting